src/scripts/script.js

```diff
@@ -38,179 +42,200 @@ const getCurrentAdditionalTemplate = () => {
+  localStorage.removeItem(+key);
+  setCountTodos();
+};
 
 const editTodoTask = (selectedNode) => {
+  $("#exampleModal").modal("show");
+  getElementById("exampleModalLabel").innerText = "Edit task";
 
-    $('#exampleModal').modal("show");
-    getElementById("exampleModalLabel").innerText = "Edit task";
-
-    const btn_editTask = getElementById("form-group-save-task");
-    btn_editTask.innerText = "Edit task";
+  const btn_editTask = getElementById("form-group-save-task");
+  btn_editTask.innerText = "Edit task";
 
-    const node_id = +selectedNode.parentNode.parentNode.parentNode.id;
-    const { id, title, text, radio: priority, isCompleted } = JSON.parse(localStorage.getItem(node_id));
+  const node_id = +selectedNode.closest(".currentTodo").id;
+  const { id, title, text, radio: priority, isCompleted } = JSON.parse(
+    localStorage.getItem(node_id)
+  );
 
-    setModalValues(title, text, priority);
+  setModalValues(title, text, priority);
 
-    btn_editTask.addEventListener("click", () => {
-        localStorage.removeItem(id);
-        addTodoToLocalStorage(id, id, ...getModalValues(), isCompleted);
-    })
-}
+  btn_editTask.addEventListener("click", () => {
+    localStorage.removeItem(id);
+    addTodoToLocalStorage(id, id, ...getModalValues(), isCompleted);
+  });
+};
 const completeTodoTask = (selectedNode) => {
-    const node_id = +selectedNode.parentNode.parentNode.parentNode.id;
-
-    const { id, title, text, radio: priority, isCompleted } = JSON.parse(localStorage.getItem(node_id));
-    localStorage.removeItem(+node_id);
-    addTodoToLocalStorage(node_id, id, title, text, priority, !isCompleted);
-    location.reload();
-
-}
+  const node_id = +selectedNode.parentNode.parentNode.parentNode.id;
+
+  const { id, title, text, radio: priority, isCompleted } = JSON.parse(
+    localStorage.getItem(node_id)
+  );
+  localStorage.removeItem(+node_id);
+  addTodoToLocalStorage(node_id, id, title, text, priority, !isCompleted);
+  location.reload();
+};
 
 const getModalValues = () => {
-    const input_titleValue = getElementById("inputTitle").value;
-    const input_textValue = getElementById("inputText").value;
-    const input_radioValue = document.querySelector('input[name="gridRadios"]:checked').value;
-    return [input_titleValue, input_textValue, input_radioValue];
-}
+  const input_titleValue = getElementById("inputTitle").value;
+  const input_textValue = getElementById("inputText").value;
+  const input_radioValue = document.querySelector(
+    'input[name="gridRadios"]:checked'
+  ).value;
+  return [input_titleValue, input_textValue, input_radioValue];
+};
 const setModalValues = (title, text, priority) => {
-    getElementById("inputTitle").value = title;
-    getElementById("inputText").value = text;
-    getElementById(`${priority}`).setAttribute("checked", "checked");
-}
+  getElementById("inputTitle").value = title;
+  getElementById("inputText").value = text;
```

Comment:
you have this variable in several places. think how to initialize them once and reuse in `getModalValues `, `setModalValues ` and etc.

Reply:
getModalValues  setModalValues this are functions and they are using in several places!
Maybe i didn't correct understand you. Did you mean getElementById("inputTitle").value = title;  getElementById("inputText").value = text???